src/components/MovieCard/movieCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './movieCard.scss';
+
+export const MovieCard = ({ movie }) => (
+  <div className="movie-card">
+    <img
+      className="movie-card__image"
+      src={movie.Poster}
```

Comment:
Not an error, but usually keys of object shouldn’t start with upper letter.
```js
movie.poster
```

Reply:
fixed, thanks
it was just copy/paste from omdbapi)